### PR TITLE
Assorted mappings and a bunch of param names

### PR DESCRIPTION
**Pull request by i509VCB**

### CODE REVIEW

File: mappings/net/minecraft/item/ArmorMaterials.mapping [hunk 1]
```diff
@@ -17,3 +17,4 @@ CLASS net/minecraft/class_1740 net/minecraft/item/ArmorMaterials
 		ARG 7 equipSound
 		ARG 8 toughness
 		ARG 9 knockbackResistance
+		ARG 10 ingredient
```
Comment (liach) [inline mappings/net/minecraft/item/ArmorMaterials.mapping]: this is really repair ingredient

Comment (YanisBft) [inline mappings/net/minecraft/item/ArmorMaterials.mapping]:
```suggestion
		ARG 10 repairIngredientSupplier
```

**Comment by liach** [comment]
Also @i509VCB mind change branch to merge into

**Comment by i509VCB** [comment]
done

**Comment by i509VCB** [comment]
oops

**Review by YanisBft — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Review by Gegy — APPROVED** [review]

**Pull request merged**